gradio/helpers.py

```diff
@@ -797,3 +806,24 @@ def get_color_gradient(c1, c2, n):
 
     subprocess.call(ffmpeg_cmd, shell=True)
     return output_mp4.name
+
+
+@document()
+class EventData:
```

Comment:
This part of the docs is still confusing for me: 

<img width="794" alt="image" src="https://user-images.githubusercontent.com/1778297/224849530-a2c02f3b-60f8-41b8-8887-7d736283604b.png">

It says that the attributes of `EventData` are `target` and `data` but the example shows the subclass `gr.SelectData` having attributes `index`.

Reply:
We should also document the target and data parameters!

Reply:
well EventData only has target, but the subclass carries extra attributes. Will try to clarify but feel free to edit.

Reply:
Can we document the subclasses too then? Nowhere do we document that `SelectData` has `index` for example